cloudformation/story-packages.json

```diff
@@ -308,6 +329,186 @@
                 }
             }
         },
+        "DynamoDBDataPipeline": {
+            "Type": "AWS::DataPipeline::Pipeline",
```

Comment:
Just translating [this concept](http://docs.amazonaws.cn/en_us/amazondynamodb/latest/developerguide/DynamoDBPipeline.html) in cloudformation.